dev/io.openliberty.jakarta.mail.2.0/src/io/openliberty/jakarta/mail/internal/MailSessionService.java

```diff
@@ -0,0 +1,297 @@
+/*******************************************************************************
+ * Copyright (c) 2014, 2015, 2017, 2020 IBM Corporation and others.
```

Comment:
Only 2, first and last

Reply:
Aside from changes from javax to jakarta, how much of this code is shared with the previous versions? Can you make it common code and transform it for jakarta?